test/bindings/http/unmarshaller_0_3_tests.js

```diff
@@ -110,8 +110,12 @@ describe("HTTP Transport Binding Unmarshaller for CloudEvents v0.3", () => {
       // act and assert
       un.unmarshall(payload, headers)
         .then(() => { throw new Error("failed"); })
-        .catch((err) =>
-          expect(err.message).to.equal("invalid payload"));
+        .catch((err) => {
+          expect(err.message).to.equal("invalid payload");
+        })
+        .catch((err) => {
+          console.error(err);
```

Comment:
I think this should fail the test. If the message is not correct, it will throw. This seems to indicate that there's something wrong with the test itself. It's failing but not registering as a failure.

Reply:
PR updated, please take a look when you have a time. :+1: